src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    public Square(int area, Color color) {
```

Comment:
let's add field "side" that would be passed from constructor

Reply:
done